p2panda-js/src/wasm-adapter/index.ts

```diff
@@ -6,7 +6,7 @@
 declare const BUILD_TARGET_WEB: boolean;
 
 const adapter = BUILD_TARGET_WEB
-  ? (require('~/wasm-adapter/browser') as typeof import('./browser'))
-  : (require('~/wasm-adapter/node') as typeof import('./node'));
+  ? (require('~/wasm-adapter/browser') as typeof import('~/wasm-adapter/browser'))
+  : (require('~/wasm-adapter/node') as typeof import('~/wasm-adapter/node'));
```

Comment:
Why do you prefer this?

Reply:
We introduced aliases, either we use them or remove aliases at all :-D

Reply:
I prefer using whichever form is shorter / easier to parse. I would take the alias over `../../../..` any time but wouldn't use it if we are just descending one directory.